### PR TITLE
add shipping method ups freight for get shipping rates

### PR DESCRIPTION
**Pull request by TusharSangani**
add shipping method ups freight for get shipping rates

### CODE REVIEW

File: Libraries/Hotcakes.Commerce.NoCms/Hotcakes.Commerce.NoCms.csproj [hunk 1]
```diff
@@ -58,9 +58,8 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\References\Dnn\System.Web.Helpers.dll</HintPath>
     </Reference>
-    <Reference Include="System.Web.Mvc, Version=4.0.0.1, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\packages\Microsoft.AspNet.Mvc.4.0.40804.0\lib\net40\System.Web.Mvc.dll</HintPath>
+    <Reference Include="System.Web.Mvc">
```
Comment (hismightiness) [inline Libraries/Hotcakes.Commerce.NoCms/Hotcakes.Commerce.NoCms.csproj]: Why was the MVC upgrade necessary?

File: Libraries/Hotcakes.Commerce/Hotcakes.Commerce.csproj [hunk 2]
```diff
@@ -123,9 +123,8 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\References\Dnn\System.Web.Helpers.dll</HintPath>
     </Reference>
-    <Reference Include="System.Web.Mvc, Version=4.0.0.1, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\packages\Microsoft.AspNet.Mvc.4.0.40804.0\lib\net40\System.Web.Mvc.dll</HintPath>
+    <Reference Include="System.Web.Mvc">
```
Comment (hismightiness) [inline Libraries/Hotcakes.Commerce/Hotcakes.Commerce.csproj]: Why was the MVC upgrade necessary?

File: Libraries/Hotcakes.Shipping.UpsFreight/BillingOption.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
```
Comment (hismightiness) [inline Libraries/Hotcakes.Shipping.UpsFreight/BillingOption.cs]: This is missing the open source license header

File: Libraries/Hotcakes.Shipping.UpsFreight/HandlineOneUnitType.cs [hunk 4]
```diff
@@ -0,0 +1,21 @@
+﻿using System;
```
Comment (hismightiness) [inline Libraries/Hotcakes.Shipping.UpsFreight/HandlineOneUnitType.cs]: This is missing the open source license header

File: Libraries/Hotcakes.Shipping.UpsFreight/PackingTypes.cs [hunk 5]
```diff
@@ -0,0 +1,52 @@
+﻿using System;
```
Comment (hismightiness) [inline Libraries/Hotcakes.Shipping.UpsFreight/PackingTypes.cs]: This is missing the open source license header

File: Libraries/Hotcakes.Shipping.UpsFreight/Properties/AssemblyInfo.cs [hunk 6]
```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
```
Comment (hismightiness) [inline Libraries/Hotcakes.Shipping.UpsFreight/Properties/AssemblyInfo.cs]: This is missing the open source license header

File: Libraries/Hotcakes.Shipping.UpsFreight/ServiceTypes.cs [hunk 7]
```diff
@@ -0,0 +1,21 @@
+﻿using System;
```
Comment (hismightiness) [inline Libraries/Hotcakes.Shipping.UpsFreight/ServiceTypes.cs]: This is missing the open source license header

File: Website/DesktopModules/Hotcakes/Core/Admin/Parts/Shipping/UPSFreight/App_LocalResources/Edit.ascx.resx [hunk 8]
```diff
@@ -0,0 +1,408 @@
...
+  <data name="rfvAdjustmentTextBox.ErrorMessage" xml:space="preserve">
+    <value>Adjustment price is required.</value>
+  </data>
+  <data name="ServicesToUse.Text" xml:space="preserve">
+    <value>Service(s) to Use</value>
+  </data>
+  <data name="ShippingZone.Text" xml:space="preserve">
+    <value>Shipping Zone</value>
+  </data>
+  <data name="UPS10KgBox.Text" xml:space="preserve">
```
Comment (hismightiness) [inline Website/DesktopModules/Hotcakes/Core/Admin/Parts/Shipping/UPSFreight/App_LocalResources/Edit.ascx.resx]: Are these standard UPS package and shipping keys/values necessary for the freight shipping method?

File: Libraries/Hotcakes.Commerce.NoCms/Hotcakes.Commerce.NoCms.csproj (hunk 1 above)
Comment (TusharSangani) [reply Libraries/Hotcakes.Commerce.NoCms/Hotcakes.Commerce.NoCms.csproj]: i have reverted above change, when i have checkout project and try to build but it failed due to System.Web.Mvc.dll was missing so,  i have used  reference from nuget packages folder has latest version of MVC DLL(Microsoft.AspNet.Mvc.5.1.1) so, i have added reference of same.

File: Libraries/Hotcakes.Commerce/Hotcakes.Commerce.csproj (hunk 2 above)
Comment (TusharSangani) [reply Libraries/Hotcakes.Commerce/Hotcakes.Commerce.csproj]: i have reverted above change, when i have checkout project and try to build but it failed due to System.Web.Mvc.dll was missing so,  i have used  reference from nuget packages folder has latest version of MVC DLL(Microsoft.AspNet.Mvc.5.1.1) so, i have added reference of same.

File: Libraries/Hotcakes.Shipping.UpsFreight/BillingOption.cs (hunk 3 above)
Comment (TusharSangani) [reply Libraries/Hotcakes.Shipping.UpsFreight/BillingOption.cs]: done. added license header.

File: Libraries/Hotcakes.Shipping.UpsFreight/HandlineOneUnitType.cs (hunk 4 above)
Comment (TusharSangani) [reply Libraries/Hotcakes.Shipping.UpsFreight/HandlineOneUnitType.cs]: done. added license header.

File: Libraries/Hotcakes.Shipping.UpsFreight/PackingTypes.cs (hunk 5 above)
Comment (TusharSangani) [reply Libraries/Hotcakes.Shipping.UpsFreight/PackingTypes.cs]: done. added license header.

File: Libraries/Hotcakes.Shipping.UpsFreight/Properties/AssemblyInfo.cs (hunk 6 above)
Comment (TusharSangani) [reply Libraries/Hotcakes.Shipping.UpsFreight/Properties/AssemblyInfo.cs]: done,added license header.

File: Libraries/Hotcakes.Shipping.UpsFreight/ServiceTypes.cs (hunk 7 above)
Comment (TusharSangani) [reply Libraries/Hotcakes.Shipping.UpsFreight/ServiceTypes.cs]: done,added license header.

File: Website/DesktopModules/Hotcakes/Core/Admin/Parts/Shipping/UPSFreight/App_LocalResources/Edit.ascx.resx (hunk 8 above)
Comment (TusharSangani) [reply Website/DesktopModules/Hotcakes/Core/Admin/Parts/Shipping/UPSFreight/App_LocalResources/Edit.ascx.resx]: i have re-verify all keys and removed which are not used for freight shipping method.

**Pull request merged**